core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -709,7 +709,7 @@ class BrokerServer(
         None
       }
 
-      val rlm = new RemoteLogManager(config.remoteLogManagerConfig, config.brokerId, config.logDirs.head, clusterId, time,
+      val rlm = new RemoteLogManager(config.remoteLogManagerConfig, config.brokerId, config.logDirs.asScala.head, clusterId, time,
```

Comment:
It looks like we can use `getFirst` to avoid redundant conversion.

```suggestion
      val rlm = new RemoteLogManager(config.remoteLogManagerConfig, config.brokerId, config.logDirs.getFirst, clusterId, time,
```

Reply:
Thanks. I’m using get(0) instead, since List.getFirst() is available starting with Java 21.